### PR TITLE
Add suport to combined mouse buttons

### LINKED ISSUE #6: Mouse: add support for multiple buttons pressed at the same time
See https://github.com/crystal-bit/input-overlay/blob/main/addons/input-overlay/overlay-widget/overlay-widget.gd#L16

### PR DESCRIPTION
**Pull request by lucashilles**
closes #6

### CODE REVIEW

**Comment by davcri** [comment]
thanks for the PR @lucashilles 
I'm reviewing it right now

File: addons/input-overlay/assets/Mouse_Left_Key.png.import
Comment (davcri) [inline addons/input-overlay/assets/Mouse_Left_Key.png.import:1]: The name change is fine to me, however having these kind of changes in a separate commit makes it easier to review for other people.

Fine for this PR though :)

File: addons/input-overlay/overlay-widget/mouse-buttons.gd [hunk 1]
```diff
@@ -1,19 +1,17 @@
...
 
+var button_overlays = {}
 
-@export var lmb: Texture
-@export var rmb: Texture
-@export var mmb: Texture
-@export var idle: Texture
+@onready var lmb = $LMB
+@onready var mmb = $MMB
+@onready var rmb = $RMB
```
Comment (davcri) [inline addons/input-overlay/overlay-widget/mouse-buttons.gd:8]:
```
@onready var button_overlays = {
	lmb = $LMB,
	mmb = $MMB,
	rmb = $RMB
}
```

this is also valid GDScript :)

File: addons/input-overlay/overlay-widget/overlay-widget.gd [hunk 2]
```diff
@@ -1,39 +1,45 @@
...
+	else:
+		# For some reason, the event fired on button release only reports the
+		# released button
+		if button_index == MOUSE_BUTTON_LEFT:
+			buttons = { "lmb": false }
+		elif button_index == MOUSE_BUTTON_MIDDLE:
+			buttons = { "mmb": false }
+		elif button_index == MOUSE_BUTTON_RIGHT:
+			buttons = { "rmb": false }
+	return buttons
```
Comment (davcri) [inline addons/input-overlay/overlay-widget/overlay-widget.gd:42]: Since this is related to the mouse I would move it in the `mouse-buttons.gd` script.

**Review by davcri — APPROVED** [review]
PR fine to me, thanks for taking time to work on it.
Sure it's a welcome upgrade for every user of this addon 🎉 

I left some comment in the review, just some random idea for future improvements. If you want to create a followup PR feel free to do it!

File: addons/input-overlay/overlay-widget/mouse-buttons.gd (hunk 1 above)
Comment (lucashilles) [reply addons/input-overlay/overlay-widget/mouse-buttons.gd:8]: That's nice, I'll apply to the next PR

File: addons/input-overlay/overlay-widget/overlay-widget.gd (hunk 2 above)
Comment (lucashilles) [reply addons/input-overlay/overlay-widget/overlay-widget.gd:42]: Yeah, probably its better to put there, I'll fix on the next PR

**Pull request merged**